:bug: Arregla pruebas

AlejandroPad opened the pull request:
Se cambio la imagen madre por `base:0.7.0`. Igualmente se arreglo un error en src/01_create_... en el cual se tomaba al sexo de las especies como longitud.

- [ ] Borrar .devcontainer.json

AlejandroPad committed:
se cambio la linea 1 del DockerFile por FROM islasgeci/base:0.7.0

AlejandroPad committed:
:bug: Se cambio la imagen madre  y se arreglo un error

islasfernando commented:
El :octocat: no nos dejaba consignar por eso consignamos de más 🤡, @AlejandroPad hay que borrar también los archivos:

- [ ] Todos los archivos de `data/processed/`

AlejandroPad committed:
Update 01_create_parameter_logistic_model_LAAL.R

AlejandroPad committed:
Se realizo rm --force --recursive data/processed
rm --force --recursive dimorfismo.Rcheck
rm --force --recursive man
rm --force --recursive reports/pythontex*
rm --force *.tar.gz
rm --force reports/*.aux
rm --force reports/*.log
rm --force reports/*.pdf
rm --force reports/*.pytxcode
rm --r data/raw/carpeta1

devarops commented:
Muchas gracias pot tu primer PR. 🙌🏾 
Seguiremos trabajando en el otro PR #37
